pkg/controllers/job/job_controller_util.go

```diff
@@ -57,6 +57,10 @@ func createJobPod(job *batch.Job, template *v1.PodTemplateSpec, ix int) *v1.Pod
 	if len(pod.Spec.SchedulerName) == 0 {
 		pod.Spec.SchedulerName = job.Spec.SchedulerName
 	}
+	// If dns policy not set when hostNetwork=true, set it to `ClusterFirstWithHostNet`
+	if pod.Spec.HostNetwork == true && pod.Spec.DNSPolicy == "" {
```

Comment:
can we add this into admission controller?

Reply:
yes, anyway makes sense to me